src/ClientInterface.php

```diff
@@ -89,6 +91,8 @@
  * @method int               decr(string $key)
  * @method int               decrby(string $key, int $decrement)
  * @method Status            failover(?To $to = null, bool $abort = false, int $timeout = -1)
+ * @method Status            ftcreate(string $index, CreateArguments $arguments, Schema $schema)
```

Comment:
`CreateArguments $arguments` should be optional

Reply:
Oh, you're right 👌

Reply:
Fixed